shapeout/lin_mix_mod.py

```diff
@@ -16,6 +16,36 @@
 
```

Comment:
Since appveyor will not let us test your pull request, could you please run
`python ShapeOut/tests/test_linmixmod.py` and see if the test passes?